zdrive/download/downloader.py

```diff
@@ -9,6 +9,8 @@
 from googleapiclient.http import MediaIoBaseDownload
 from zdrive import DriveAPI
 from multiprocessing import Process
+from signal import signal, SIGPIPE, SIG_DFL
+signal(SIGPIPE, SIG_DFL)
```

Comment:
This fixes the following error. I assume it occurs when there's a large number of files to be downloaded. 

```
IOError: [Errno 32] Broken pipe
```

Ref: https://stackoverflow.com/questions/14207708/ioerror-errno-32-broken-pipe-when-piping-prog-py-othercmd

Reply:
this is quite helpful and definitely a better implementation for the purpose. Thanks @maskaravivek